Makefile

```diff
@@ -56,6 +62,11 @@ build-cli-linux: syncthing-linux certs
 	rice append -i ./pkg/syncthing --exec blimp-linux
 	rm ./pkg/syncthing/stbin
 
+build-cli-windows: syncthing-windows.exe certs
```

Comment:
Is connecting to the Docker daemon nicer when using the native build?

Reply:
Yes, it just works. The library knows to connect to the right named pipe.

Reply:
Wonderful